system-tests/test/component_testing_spec.ts

```diff
@@ -122,6 +122,14 @@ describe(`Angular CLI major versions`, () => {
       browser: 'chrome',
       expectedExitCode: 0,
     })
+
+    systemTests.it(`v${majorVersion} Zone.JS does not break Mocha`, {
```

Comment:
Rather than run these scaffolding twice, I say we edit the spec for the test above to include these tests so:
```ts
for (const majorVersion of ANGULAR_MAJOR_VERSIONS) {
  let spec = 'src/**/*.cy.ts'

  if (majorVersion === '13') {
    spec = `${spec},!src/app/components/standalone.component.cy.ts`
  }

  systemTests.it(`v${majorVersion} with mount tests`, {
    project: `angular-${majorVersion}`,
    spec,
    testingType: 'component',
    browser: 'chrome',
    expectedExitCode: 0,
  })
}
```

These tests are expensive so if we can include them in another test I think it's a win.